example/web/index.html

```diff
@@ -31,74 +31,28 @@
 
   <title>example</title>
   <link rel="manifest" href="manifest.json">
+
+  <script>
```

Comment:
This is the new index.html page for Flutter 3.

https://docs.flutter.dev/development/platform-integration/web/initialization#getting-started